tests/test_jpeg.py

```diff
@@ -0,0 +1,86 @@
+from PIL import Image
+import os
+import urllib.request
+from fastapi import APIRouter
+from pims.formats import FORMATS
+import io
+from pims.importer.importer import FileImporter
+from tests.utils.formats import info_test, thumb_test, resized_test, mask_test, crop_test, crop_null_annot_test, histogram_perimage_test
+import pytest
+from pims.files.file import (
+    EXTRACTED_DIR, HISTOGRAM_STEM, ORIGINAL_STEM, PROCESSED_DIR, Path,
+    SPATIAL_STEM, UPLOAD_DIR_PREFIX, unique_name_generator
+)
+
+def get_image(path, image):
```

Comment:
This function has many duplicate code fragments between the different format tests. What do you think about making this function generic regarding the format, to avoid all these copy/paste ?
Something like `get_image(path, image, download_url, upload_dir_builder_fn)` where `upload_dir_builder_fn` is a function that build the upload_dir, processed_dir, make the visualisation file etc ? This is the only code fragment that change from one format to another, no ?